Building problem on Appveyor/Python 3.5/setuptools 24.*/25.*

In the linking process, ```/LIBPATH``` are not quoted. As result path containing spaces are interpreted as object files:

* ```/LIBPATH:C:Program Files...``` became:
* ```LINK : fatal error LNK1181: cannot open input file 'Files.obj'```

I open a bug there
- https://github.com/pypa/setuptools/issues/694

But maybe it is also relative to our setup.py, cause ```silx``` build without any problem on the same environment.

I only force to use ```setuptools==23``` on appveyor configuration file.

--- a/setup.py
+++ b/setup.py
@@ -28,7 +28,7 @@ __author__ = "Jerome Kieffer"
 __contact__ = "Jerome.Kieffer@ESRF.eu"
 __license__ = "GPLv3+"
 __copyright__ = "European Synchrotron Radiation Facility, Grenoble, France"
-__date__ = "04/08/2016"
+__date__ = "24/08/2016"
 __status__ = "stable"
 
 install_warning = True
@@ -56,12 +56,7 @@ except ImportError:
     from distutils.command.install import install
     from distutils.command.build_py import build_py as _build_py
 
-# Fix use of Extension from numpy if it is really needed
-# Our setup is not working anymore with setuptools >=24, Python 3.5, Windows
-# Is numpy really needed for compilation? else we have to investigate
-# Silx setup looks to not have this problem. It can be interesting to reuse it
-#from numpy.distutils.core import Extension as _Extension
-from setuptools import Extension as _Extension
+from numpy.distutils.core import Extension as _Extension
 
 
 PROJECT = "pyFAI"

